common/script/roleInfo.js

```diff
@@ -5176,27 +5188,11 @@
     "localprops": [
       {
         "is": "property",
-        "name": "aria-label",
+        "name": "aria-braillelabel",
```

Comment:
git diff being confused here - paragraph gained braillelabel (disallowed) while presentation lost it's information (cf. earlier comment on "none" /  #1945).